src/libraries/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -907,6 +907,145 @@ public static async Task ZipArchive_InvalidHuffmanData()
             }
         }
 
+        [Fact]
+        public static void ZipArchive_InvalidVersionToExtract()
+        {
+            using (MemoryStream updatedStream = new MemoryStream())
+            {
+                int originalLocalVersionToExtract = s_inconsistentVersionToExtract[4];
+                int originalCentralDirectoryVersionToExtract = s_inconsistentVersionToExtract[57];
+
+                // The existing archive will have a "version to extract" of 0.0, but will contain entries
+                // with deflate compression (which has a minimum version to extract of 2.0.)
+                Assert.Equal(0x00, originalLocalVersionToExtract);
+                Assert.Equal(0x00, originalCentralDirectoryVersionToExtract);
+
+                updatedStream.Write(s_inconsistentVersionToExtract);
```

Comment:
A comment along these lines would be good:
```suggestion
                // Writes a full zip archive
                updatedStream.Write(s_inconsistentVersionToExtract);
```